pilot/cmd/pilot-discovery/main.go

```diff
@@ -96,8 +96,6 @@ func init() {
 			serviceregistry.CloudFoundryRegistry, serviceregistry.MockRegistry, serviceregistry.ConfigRegistry))
 	discoveryCmd.PersistentFlags().StringVar(&serverArgs.Config.CFConfig, "cfConfig", "",
 		"Cloud Foundry config file")
-	discoveryCmd.PersistentFlags().StringVar(&serverArgs.Config.ClusterRegistriesConfigmap, "clusterRegistriesConfigMap", "",
```

Comment:
Should we deprecate this flag first and remove it later in a future version?

Any deprecation policy do we have @rshriram ?

Reply:
The ability to pass in a command line argument to enable multicluster was deprecated a while back.  None of the 1.0 documentation refers to this approach and the code to support it has been stripped out for a while.   So i am following exactly the procedure you are advocating here :-)